.travis.yml

```diff
@@ -8,14 +8,28 @@ addons:
       - libgconf-2-4
 branches:
   only:
-    - master
-before_install:
-  - npm install -g yarn
```

Comment:
We don't need to install yarn before using it?

Reply:
yarn is present on travis already:
![image](https://user-images.githubusercontent.com/6325237/70338453-f52ffa80-181a-11ea-90ee-deef94a126e6.png)